gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -648,8 +570,24 @@ gst_tensor_filter_common_get_property (GstTensorFilterPrivate * priv,
       g_value_set_string (value, prop->fwname);
       break;
     case PROP_MODEL:
-      g_value_set_string (value, prop->model_file);
+    {
+      GString *gstr_models = g_string_new (NULL);
+      gchar *models;
+      int idx;
+
+      /* return a comma-separated string */
+      for (idx = 0; idx < prop->num_models; ++idx) {
+        if (idx != 0) {
+          g_string_append (gstr_models, ",");
+        }
+
+        g_string_append (gstr_models, prop->model_files[idx]);
+      }
+
+      models = g_string_free (gstr_models, FALSE);
+      g_value_set_string (value, models);
```

Comment:
use g_value_take_string().
g_value_set_string() will not take ownership of passed argument.
if you want to use g_value_set_string(), plz add g_free (models);

Reply:
Ohh thanks. Update.d